src/pages/admin/categories/AddCategory/index.js

```diff
@@ -1,7 +1,33 @@
 import React from 'react';
+import { Card, Button, Form } from 'react-bootstrap';
+import { BsFillArrowLeftSquareFill } from 'react-icons/bs';
+
+import style from './index.module.scss';
 
 const AddCategory = () => {
-  return <div></div>;
+  return (
+    <Card style={{ width: '1063px' }} className={style.card}>
+      <Card.Header className={style.header}>
+        <div>
+          <a href="/admin/categories">
+            <BsFillArrowLeftSquareFill className={style.backarrow} />
+          </a>
+        </div>
+        <div className={style.headerText}>
+          <span>Add Category</span>
+        </div>
+      </Card.Header>
+      <Card.Body>
+        <Form>
+          <Form.Label className={style.textTitle}>Title</Form.Label>
+          <Form.Control type="title" className={style.inputFieldTitle} />
+          <Form.Label className={style.textDescription}>Description</Form.Label>
+          <Form.Control as="textarea" className={style.inputFieldDescription} />
+        </Form>
+        <Button className={style.button}>Add Category</Button>
+      </Card.Body>
+    </Card>
+  );
 };
 
 export default AddCategory;
```

Comment:
can we make just one AddEditCategory file and pass props to indicate whether we will be adding or editing a category? we can name the file 'CategoryUpsertForm'